hosting/src/app/games/quiz/lobby/lobby.component.html

```diff
@@ -65,8 +65,8 @@ <h2>Quiz Overview</h2>
                 <div class="round-list__item">
                     <brand-expansion label="{{ round.title }} ({{ round.questionList | count }})" (onToggle)="loadQuestions(round)">
                         <ng-container *ngIf="round.questions; else noQuestions">
-                            <div class="question" *ngFor="let question of round.questions">
-                                <div>{{ question.text | slice: 0 : 30 }}</div>
+                            <div class="question" *ngFor="let question of round.questions; index as i">
+                                <div><span class="qNumber">{{i+1}}</span> {{question.text | slice: 0 : 29 }}</div>
```

Comment:
Class name to be hyphenated and spaces around braces ...

```suggestion
                                <div><span class="question-number">{{ i+1 }}</span> {{ question.text | slice: 0 : 29 }}</div>
```